draft-jhoyla-req-mtls-flag.md

```diff
@@ -133,7 +133,56 @@ TLS namespace with the following values:
 
 --- back
 
+# Appendix
+
+## Alternative Approaches
+
+### Concealed Auth
+
+Concealed Auth {{?RFC9729}} is a draft that allows a client to attach a header
+at the HTTP layer that cryptographically binds a public key to the underlying
+TLS connection. This gives us a similar authentication property to using
+`request_client_auth`, and is also client initiated. This solution is applicable
+in some contexts, but in the case of authenticating bots is difficult to
```

Comment:
```suggestion
in some contexts, but in the case of authenticating bots may be difficult to
```

Reply:
Also, why would this be hard to implement specifically for the bots use case and not others?